src/pcapi/models/offerer.py

```diff
@@ -42,13 +42,12 @@ class Offerer(
         String(9), nullable=True, unique=True
     )  # FIXME: should not be nullable, is until we have all SIRENs filled in the DB
 
-    def give_rights(self, user, rights):
+    def give_rights(self, user):
```

Comment:
On peux surement renommer ça au passage

```suggestion
    def grant_access(self, user):
```

Reply:
J'aime beaucoup l'idée de renommer en `grant_access()` et `has_access()`. Bien vu, je prends !